project.clj

```diff
@@ -1,33 +1,43 @@
-(defproject one-time "0.7.0"
+(defproject one-time "0.7.1"
   :author "Suvash Thapaliya"
   :description "One Time Password (TOTP and HOTP) library for Clojure. TOTP/HOTP is widely used for Two factor / Multi Factor Authentication."
   :url "http://github.com/suvash/one-time"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [commons-codec "1.14"]
-                 [ring/ring-codec "1.1.2"]
-                 [com.github.kenglxn.qrgen/javase "2.6.0"]]
-  :profiles {:dev {:plugins      [[lein-cloverage "1.1.2"]
-                                  [lein-codox "0.10.7"]
-                                  [lein-ancient "0.6.15"]]}
+  :dependencies [[org.clojure/clojure "1.11.1"]
+                 [commons-codec "1.15"]
+                 [ring/ring-codec "1.2.0"]
+                 [com.google.zxing/javase "3.5.0"]
+                 [org.apache.xmlgraphics/batik-dom "1.15"]
+                 [org.apache.xmlgraphics/batik-svggen "1.15"]
+                 [com.github.kenglxn.qrgen/javase "2.6.0" :exclusions [com.google.zxing/javase
+                                                                       org.apache.xmlgraphics/batik-dom
+                                                                       org.apache.xmlgraphics/batik-svggen]]]
+  :plugins      [[lein-ancient "1.0.0-RC4-SNAPSHOT"]]
+  :profiles {:dev {:plugins      [[lein-cloverage "1.2.4"]
+                                  [lein-codox "0.10.8"]
+                                  [lein-ancient "1.0.0-RC4-SNAPSHOT"]]}
              :1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}
```

Comment:
which probably means that I could drop support for older unsupported versions finally.